backend-project/docs/modules/files.rst

```diff
@@ -0,0 +1,19 @@
+Pliki
+=====
+
+Założenia
+---------
+
+Moduł odpowiedzialny za informację o plikach powiązanych z listami.
+
+--------------------------------------------------------------------
```

Comment:
Myślę, że w całej tej dokumentacji takiej linie nie są konieczne.

Reply:
Hmm... to bardziej kwestia estetyczna. Dla mnie takie oddzielanie sekcji jest czytelniejsze, zwłaszcza jak tekstu jest dużo, ale jeżeli uważasz, że powinnam to usunąć, to poprawię.
<img width="747" alt="oddzielnie-sekcji" src="https://user-images.githubusercontent.com/29001525/98528274-5eff2280-227c-11eb-81b9-d9adad15769f.png">

Reply:
Kwestie estetyczne powinny być rozstrzygane globalnie w stylu, a nie wymuszane przez lokalne formatowanie. Obawiam się, że w innym przypadku za chwilę zaczniemy gubić linię i będziemy porządkować, aby ujednolicić brakujące linie.

Reply:
Ok. Ten argument do mnie przemawia :) Zaraz zrobię poprawkę.